.github/workflows/rebuild-clade-i.yaml

```diff
@@ -0,0 +1,60 @@
+name: Rebuild clade-I
+
+on:
+  repository_dispatch:
+    types:
+      - rebuild
+      - rebuild_clade-i
+
+  workflow_dispatch:
+    inputs:
+      trial_name:
+        description: "If set, result will be at nextstrain.org/staging/trial/${trial_name}/${auspice_name}"
+        required: false
+
+jobs:
+  set_config_overrides:
+    runs-on: ubuntu-latest
+    steps:
+      - id: config
+        name: Set config overrides
+        env:
+          TRIAL_NAME: ${{ inputs.trial_name }}
+        run: |
+          config=""
+          if [[ "$TRIAL_NAME" ]]; then
+            config+="--config"
+            config+=" deploy_url='s3://nextstrain-staging/'"
+            config+=" auspice_prefix='"$TRIAL_NAME"'"
+          fi
+
+          echo "config=$config" >> "$GITHUB_OUTPUT"
+    outputs:
+      config_overrides: ${{ steps.config.outputs.config }}
+
+  rebuild_clade_i:
+    needs: [set_config_overrides]
+    permissions:
+      id-token: write
+    uses: nextstrain/.github/.github/workflows/pathogen-repo-build.yaml@master
+    secrets: inherit
+    with:
+      runtime: conda
+      env: |
+        CONFIG_OVERRIDES: ${{ needs.set_config_overrides.outputs.config_overrides }}
+        GITHUB_RUN_ID: ${{ github.run_id }}
+        SLACK_CHANNELS: ${{ inputs.trial_name && vars.TEST_SLACK_CHANNEL || vars.SLACK_CHANNELS }}
+        BUILD_DIR: phylogenetic
+        BUILD_NAME: clade-i
+      run: |
+        nextstrain build \
+          --detach \
+          --no-download \
+          --cpus 2 \
+          --env GITHUB_RUN_ID \
+          --env SLACK_TOKEN \
+          --env SLACK_CHANNELS \
+          . \
+            notify_on_deploy \
+            --configfiles $BUILD_DIR/defaults/$BUILD_NAME/config.yaml $BUILD_DIR/build-configs/nextstrain-automation/config.yaml \
+            $CONFIG_OVERRIDES --directory $BUILD_DIR --snakefile $BUILD_DIR/Snakefile
```

Comment:
_non-blocking_

This can all get simplified now that we have a top level `nextstrain-pathogen.yaml`, but it will require some amount of thinking through the changes in https://github.com/nextstrain/mpox/commit/1871170cfb330d4e7b75af3002e0325816d53303 and updating the other GH Action workflows as well. Totally fine to keep this PR scoped to just changes for Clade-I and I'll do updates across the GH Action workflows later.

Reply:
Tracking in https://github.com/nextstrain/mpox/issues/259

Reply:
Good background, thanks!

Let's keep this action the same as the others and fix them all in a separate push.

Reply:
> I'll do updates across the GH Action workflows later

Doesn't have to be you, by the way - I'm sure many of us can help out here.